engine/flink/util/src/main/scala/pl/touk/nussknacker/engine/flink/util/exception/DefaultEspExceptionHandler.scala

```diff
@@ -62,11 +64,22 @@ case class VerboselyLoggingRestartingExceptionHandler(processMetaData: MetaData,
 case class VerboselyLoggingExceptionConsumer(processMetaData: MetaData, params: Map[String, String] = Map.empty)
   extends FlinkEspExceptionConsumer
     with LazyLogging {
-  override def consume(e: EspExceptionInfo[NonTransientException]) = {
+  override def consume(e: EspExceptionInfo[NonTransientException]): Unit = {
     logger.error(s"${processMetaData.id}: Exception during processing job, params: $params, context: ${e.context}", e.throwable)
   }
 }
 
+case class BrieflyLoggingExceptionConsumer(processMetaData: MetaData, params: Map[String, String] = Map.empty)
```

Comment:
I can't see usage of `params`

Reply:
true that, I didn't copy/paste with enough care ;)